cl/lib/pacer.py

```diff
@@ -193,14 +195,14 @@ def get_blocked_status(docket: Docket, count_override: int | None = None):
     if count_override is not None:
         count = count_override
     elif docket.pk:
-        count = docket.docket_entries.all().count()
+        count = await docket.docket_entries.all().acount()
     else:
         count = 0
     small_case = count <= bankruptcy_privacy_threshold
-    bankruptcy_court = (
-        docket.court.jurisdiction in Court.BANKRUPTCY_JURISDICTIONS
+    bankruptcy_court = Docket.objects.filter(
+        pk=docket.pk, court__jurisdiction__in=Court.BANKRUPTCY_JURISDICTIONS
     )
```

Comment:
Hm, this tweaks a query against the Court table into one against the Docket table (which has 60M rows).

How about:

```suggestion
    bankruptcy_court = (
        await Court.objects.aget(pk=docket.court_id).jurisdiction in Court.BANKRUPTCY_JURISDICTIONS
    )
```

Reply:
Looks like we can further simplify this to:
```python
    bankruptcy_court = await Court.objects.filter(
        pk=docket.court_id, jurisdiction__in=Court.BANKRUPTCY_JURISDICTIONS
    ).aexists()
`